docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos:7
+MAINTAINER Apache BookKeeper <dev@bookkeeper.apache.org>
+
+ARG BK_VERSION=4.4.0
+ARG DISTRO_NAME=bookkeeper-server-${BK_VERSION}-bin
+ARG ZK_VERSION=3.5.2-alpha
```

Comment:
the zookeeper dependency is already there in bookkeeper package. 

you can run "bin/bookkeeper org.apache.zookeeper.ZooKeeperMain". it is same as 'bin/zkCli.sh'